src/main/java/uhtredragnarson/UhtredRagnarson.java

```diff
@@ -0,0 +1,47 @@
+package uhtredragnarson;
+
+import java.io.IOException;
+
+/**
+ * UhtredRagnarson is a chat bot that help users to manage their to-dos, deadlines and events.
+ */
+public class UhtredRagnarson {
+
+    private final Storage storage;
```

Comment:
Shouldn't these variables be in uppercase? I think the IntelliJ detects it as a warning but we probably have to follow the convention! 😄